config/sitemap.rb

```diff
@@ -0,0 +1,44 @@
+# Set the host name for URL creation
+SitemapGenerator::Sitemap.default_host = "https://railsdevs.com"
+
+SitemapGenerator::Sitemap.create do
+  # Put links creation logic here.
+  #
+  # The root path '/' and sitemap index file are added automatically for you.
+  # Links are added to the Sitemap in the order they are specified.
+  #
+  # Usage: add(path, options={})
+  #        (default options are used if you don't specify)
+  #
+  # Defaults: :priority => 0.5, :changefreq => 'weekly',
+  #           :lastmod => Time.now, :host => default_host
+  #
+  # Examples:
+  #
+  # Add '/articles'
+  #
+  #   add articles_path, :priority => 0.7, :changefreq => 'daily'
+  #
+  # Add all articles:
+  #
+  #   Article.find_each do |article|
+  #     add article_path(article), :lastmod => article.updated_at
+  #   end
+
+  add developers_path, changefreq: "always", priority: 1, lastmod: Developer.order(id: :desc).first.try(:updated_at)
```

Comment:
`Developer.maximum(:updated_at)` will return the most recently updated timestamp and can be used for the `lastmod:` option.

Reply:
Updated!